src/plugins/remark-images/plugin.ts

```diff
@@ -124,6 +124,10 @@ const plugin: Plugin = function plugin(
                     }
                 } as MdxJsxFlowElement | MdxJsxTextElement;
 
+                if ((options as any).inlineCaption) {
+                    caption.attributes.push(toJsxAttribute('className', 'inline'));
+                }
```

Comment:
I like this idea - could we add e plugin-config option to opt-out from automatically inlining empty captions (for example `preventInliningEmptyCaptions`)?
```suggestion
                const inlined = !optionsInput.preventInliningEmptyCaptions && /\s+/.test(cleanedAlt)
                if (inlined || (options as any).inlineCaption) {
                    caption.attributes.push(toJsxAttribute('className', 'inline'));
                }
```